### ISSUE TITLE
Style Page

### ISSUE THREAD

**Issue by grimario-andre** [grimario-andre/Portifolio#4]
Build the page structure using css3 and bootstrap 5.

**Comment by grimario-andre** [comment]
The choice between styling with pure CSS3, Bootstrap 5 only, or a combination of both depends on your specific preferences and needs. I will briefly explain the options to help you decide:

Pure CSS3: Styling with pure CSS3 gives you full control over your website's style. You can create custom styles, define layouts, animations, and transitions precisely. However, it may require more time and effort to create styles from scratch.

Bootstrap 5: Bootstrap 5 is a popular framework that provides a comprehensive set of pre-defined styles and components. It is designed to facilitate the development of responsive and visually appealing websites. Bootstrap 5 can speed up the styling process as you can use the classes and components provided by the framework. However, your website may end up having a similar appearance to other websites that also use Bootstrap.

Combination of both: You can also combine the use of pure CSS3 and Bootstrap 5 to get the best of both worlds. You can leverage Bootstrap's classes to quickly build a responsive framework and then customize it with pure CSS3 to add your unique touch. This allows for greater control over the style while taking advantage of Bootstrap's ease of use.

My suggestion is to start with Bootstrap 5 to streamline the styling process and ensure a responsive layout. Then, you can customize the style with pure CSS3 to make the website unique and meet your preferences.

Remember to consult the Bootstrap 5 documentation to make the most of the available classes and components. Try different combinations of styles, colors, and layouts to achieve the desired look.

**Comment by grimario-andre** [comment]
Update style on tag header

**Comment by grimario-andre** [comment]
Update style on tag header.
Fixed problema in the resolution 280 x 653px

**Comment by grimario-andre** [comment]
We added a media query rule for devices with a width smaller than 480px.
Within this rule, we adjusted the styles of the <header> to improve the visualization on smaller screens:

We reduced the padding of the <header> to padding: 10px; to save space.
We changed the direction of the <ul> list to flex-direction: column; to display it vertically.
We added a bottom margin to the <li> list items with `margin-bottom: 10px.
